pages/search/index.vue

```diff
@@ -263,6 +291,13 @@
           return { results: null, error: errorMessage, query: query.query };
         });
     },
+    mounted () {
```

Comment:
This shouldn't be needed if the toggle is wired up with the selectedFacets instead of relying on the matches.